src/Logging/DebugViewFriendlyErrorFormatter.php

```diff
@@ -127,12 +128,36 @@ public function output($statusCode)
         $output = $renderer->renderHeader();
         $output .= $renderer->renderInfo("Website Error", $this->getTitle(), $this->getBody());
 
-        if (Email::config()->admin_email) {
-            $mailto = Email::obfuscate(Email::config()->admin_email);
-            $output .= $renderer->renderParagraph('Contact an administrator: ' . $mailto . '');
+        if (!is_null($contactInfo = $this->addContactAdministratorInfo())) {
+            $output .= $renderer->renderParagraph($contactInfo);
         }
 
         $output .= $renderer->renderFooter();
         return $output;
     }
+
+    /**
+     * Generate the line with admin contact info
+     *
+     * @return ?string
+     */
+    private function addContactAdministratorInfo(): ?string
+    {
+        if (!$adminEmail = Email::config()->admin_email) {
+            return null;
+        }
+
+        if (is_string($adminEmail)) {
+            return 'Contact an administrator: ' . Email::obfuscate($adminEmail);
```

Comment:
Does this need i18n?

Reply:
that would be a minor change, I'm currently doing a patch for the bugfix :)

Reply:
Alright. I always treat missing localisation as a bug, but if it was already like that then I guess it’s fine for now :-)